modules/exploits/windows/fileformat/zahir_enterprise_plus_csv.rb

```diff
@@ -0,0 +1,73 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::Seh
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Zahir Enterprise Plus 6 Stack Buffer Overflow Exploit",
```

Comment:
`Exploit` can be removed from the name. It is listed as an exploit module.

Reply:
Will be removed on the next commit.